packages/parser-manifest/src/schema.json

```diff
@@ -534,5 +534,5 @@
     },
     "additionalProperties": false,
     "required": [],
-    "id": "Manifest"
+    "$id": "Manifest"
```

Comment:
We're automatically updating this schema now so we can't make changes here (they'll get blown away the next time we get an update).

Reply:
Ah, I see you also updated our automatic update code. Nevermind then :)